prediction/independent_poisson_model.py

```diff
@@ -46,11 +47,124 @@ def predict(self, home_team, away_team):
 
         l_A = (uA_B + vB_A) / 2.0
         l_B = (uB_A + vA_B) / 2.0
-
+        # print("l_A,l_B:",l_A,l_B)
         score_dict = self.run_simulations(l_A, l_B, self.NUM_ITERS)
-
+        # print("score_dict",score_dict)
         return score_dict
 
+
+    def getExpectation(self,home_team,away_team):
+        home_team = home_team.lower()
+        away_team = away_team.lower()
+
+        home_team_elo = get_current_elo(home_team)
+        away_team_elo = get_current_elo(away_team)
+
+        uA_B = self.fit_poisson_using_goals(
+            filter_matches(home_team),
+            home_team,
+            True
+        ).predict([away_team_elo, 1])
+
+        uB_A = self.fit_poisson_using_goals(
+            filter_matches(away_team),
+            away_team,
+            True
+        ).predict([home_team_elo, 1])
+
+        vA_B = self.fit_poisson_using_goals(
+            filter_matches(home_team),
+            home_team,
+            False
+        ).predict([away_team_elo, 1])
+
+        vB_A = self.fit_poisson_using_goals(
+            filter_matches(away_team),
+            away_team,
+            False
+        ).predict([home_team_elo, 1])
+
+        l_A = (uA_B + vB_A) / 2.0
+        l_B = (uB_A + vA_B) / 2.0
+        return [l_A,l_B]
+
+
+    def sample(self, num_iters):
+        """ Sample 
+        """
+        given_round16 = [
```

Comment:
might be better to make an abstract data class called Tournament or TournamentInput or something. take it an input